src/dnd/DragArea.jsx

```diff
@@ -10,13 +11,15 @@ class DragArea extends Component {
     lastY: 0,
     deg: 0,
     draggableId: null,
-    droppableId: null,
+    lastDroppableId: null,
     droppableContainers: [],
+    containers: this.props.containers,
```

Comment:
Remember it's setting only on initialization

If u expect change of this props you should implement `getDerivedStateFromProps` lifecycle

Do u expect it?